src/interactions/lennard_jones.jl

```diff
@@ -0,0 +1,88 @@
+"""
+    LennardJones(nl_only)
+
+The Lennard-Jones 6-12 interaction.
+"""
+struct LennardJones{C, T} <: GeneralInteraction
```

Comment:
`{T, C}` to match `T` going first elsewhere.

Reply:
I used this form to simply dispatch on `LennardJones{ShiftCutoff}` instead of `LennardJones{T, ShiftCutoff} where T`.